components/atoms/BenefitCardHeaderActive.js

```diff
@@ -68,10 +68,16 @@ export default function BenefitCardHeaderActive(props) {
             id={`${props.benefit.benefitType}-active-latestUpdates`}
             className="sm:pb-2"
           >
-            <p className="font-display sm:pb-2">{t.latestUpdate}</p>
+            <p className="font-display sm:pb-2">{t.latestStatus}</p>
             <p className="font-display font-bold text-lg">
               {props.benefit.benefitStatusProgress}
             </p>
+            <a
+              href={t.url_statusAndMessages}
+              className="text-sm mt-1 text-bright-blue-solid underline"
```

Comment:
The `text-sm` shouldn't be used (brings the font to 14px, when the min should be 16px)
```suggestion
              className="mt-1 text-bright-blue-solid underline"
```

Reply:
Fixed, thanks